### PR TITLE
Added cloud9 instructions for helper-scripts lesson

### PR DESCRIPTION
**Pull request by cjhar**
## What does this PR do and why?

Added guidance to use Cloud9 dev environment for helper-scripts lesson

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [X] Pre-commit checks passed.
- [X] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cjhar** [comment]
Ready for review, thanks!

File: content/basics/operations/helper-scripts/index.md [hunk 1]
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
...
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+  1. In the **Cloud9 terminal** navigate to `code/workspace`:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  cd cfn101-workshop/code/workspace
+  :::
+  1. Use the AWS CLI to create the stack. The required parameters `--stack-name` and `--template-body` have been pre-filled for you.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation update-stack --stack-name cfn-workshop-ec2 \
```
Comment (sdangol) [inline content/basics/operations/helper-scripts/index.md:321]:
```suggestion
  aws cloudformation update-stack --stack-name cfn-workshop-resources \
```

File: content/basics/operations/helper-scripts/index.md [hunk 2]
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
...
+  --template-body file://helper-scripts.yaml \
+  --capabilities CAPABILITY_IAM \
+  --parameters ParameterKey=AvailabilityZone,ParameterValue=YOUR-DIFFERENT-AVAILABILITY-ZONE-HERE
+  :::
+  1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-ec2/96d87030-e809-11ed-a82c-0eb19aaeb30f"
```
Comment (sdangol) [inline content/basics/operations/helper-scripts/index.md:328]:
```suggestion
  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-resources/96d87030-e809-11ed-a82c-0eb19aaeb30f"
```

File: content/basics/operations/helper-scripts/index.md [hunk 3]
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
...
+  :::
+  1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and wait for stack status to reach **UPDATE_COMPLETE**. You need to periodically select Refresh to see the latest stack status.
+::::
+
+::::tab{id="local" label="Local development"}
 1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** link in a new tab and log in to your AWS account.
 1. Click on the stack name, for example **cfn-workshop-ec2**.
```
Comment (sdangol) [inline content/basics/operations/helper-scripts/index.md:335]:
```suggestion
1. Click on the stack name, for example **cfn-workshop-resources**.
```

**Review by sdangol — CHANGES_REQUESTED** [review]
Please update the naming conventions and change the command to create the stack

**Comment by cjhar** [comment]
Thank you for review and suggestions, all incorporated and updated sample images. Note I removed the Availability Zone ID content as we don't need to trigger an initial EC2 instance replacement when we create all new resources. Estimated timing confirmed.

**Tested end2end**

File: content/basics/operations/helper-scripts/index.md [hunk 4]
```diff
@@ -278,60 +278,52 @@ notify AWS CloudFormation when all the applications are installed and configured
...
-1. On the **Review <stack_name>** page, scroll down to the bottom and tick **I acknowledge that AWS CloudFormation might create IAM resources** check box, then click on **Update stack**.
+1. On the **Review <stack_name>** page, scroll down to the bottom and tick **I acknowledge that AWS CloudFormation might create IAM resources** check box, then click on **Submit**.
+1. Wait for stack status to reach **CREATE_COMPLETE**. You need to periodically select Refresh to see the latest stack status.
+::::
+:::::
 
-    ::alert[Notice that in **Change set preview**, the _Replacement_ condition of EC2 resource is **True**. Hence, the current EC2 instance will be terminated and replaced with a new one.]{type="info"}
-
-1. You can click the **refresh** button a few times until you see in the status **UPDATE_COMPLETE**.
+5. Verify the new instance was deployed and is functional
```
Comment (sdangol) [inline content/basics/operations/helper-scripts/index.md:320]: Please put inside the tabs

File: content/basics/operations/helper-scripts/index.md [hunk 5]
```diff
@@ -381,10 +395,10 @@ You should see the AMI ID added to the page, similar to the picture below.
 
 Follow these steps to clean up created resources:
 
-1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the stack you have created in this lab. For example `cfn-workshop-ec2`.
+1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the stack you have created in this lab. For example `cfn-workshop-helper-scripts`.
 1. In the top right corner, click on **Delete**.
 1. In the pop-up window click on **Delete stack**.
```
Comment (sdangol) [inline content/basics/operations/helper-scripts/index.md:400]:
```suggestion
1. In the pop-up window click on **Delete**.
```

**Review by sdangol — CHANGES_REQUESTED** [review]
Added some more changes

**Review by sdangol — DISMISSED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**